internal/release/brew_formula_generator.sh

```diff
@@ -39,5 +39,15 @@ class Datree < Formula
   def install
     bin.install "datree"
   end
+
+  def caveats
+    <<~EOS
+      \033[32m[V] Downloaded Datree
+      [V] Finished Installation
+
+      \033[35m Usage: $ datree test [k8s-file.yaml]
+       Using Helm? => https://hub.datree.io/helm-plugin
+    EOS
```

Comment:
add here `tput init`
related to this pr: https://github.com/datreeio/datree/pull/129

Reply:
done